utils/requirements.txt

```diff
@@ -1 +1,11 @@
-
+enum
+configparser
+pylint
+pep8
+nose
+coverage
+mock
+flake8
```

Comment:
Also, it looks like the install_requires and the test_requires in setup.py no longer matches the list here.